packages/hooks-react/src/series/useEpisodes.ts

```diff
@@ -28,22 +29,26 @@ export const useEpisodes = (
 } => {
   const apiService = getModule(ApiService);
 
+  // Determine currently selected language
+  const i18n = getI18n();
+  const language = i18n.language;
```

Comment:
This hook isn't reactive on itself when the `language` changes. It might work because the parent component re-renders, but this might be considered "unstable".

Reply:
Good point, I can quickly change back to using `const { i18n } = useTranslation('menu');` instead 👍

Reply:
I reverted to using `useTranslation` hook instead 👍 
Thank you for the quick check and in-depth explanation ❤️